.github/workflows/release.yml

```diff
@@ -0,0 +1,43 @@
+#
+# Release workflow for Mount Helper Script.
+#
+
```

Comment:
I can't really review it standalone, since I don't know much about this github pipeline format.
We can  discuss this once you are done with validation